doc/de/Settings.md

```diff
@@ -238,6 +238,12 @@ Aber keine Panik!
 Friendica wird die Beiträge nicht für alle Zeiten in der Warteschlange behalten.
 Nach einiger Zeit werden Knoten als inaktiv identifiziert und Nachrichten an Nutzer dieser Knoten aus der Warteschlange gelöscht.
 
+## Server Blockliste
+
+Auf dieser Seite des Admin Panels können Administratoren einer Friendica Instanz die komplette Kommunikation (eingehdnd und ausgehend) mit bestimmten Domains unterbinden.
```

Comment:
Der Rechtschreib-Nerd in mir merkt an, dass es "Friendica-Instanz" heißen muss ;-)